Robust.Server/ServerStatus/StatusHost.Handlers.cs

```diff
@@ -48,7 +48,8 @@ private async Task<bool> HandleStatus(IStatusHandlerContext context)
                 // Content can override these if it wants (e.g. stealthmins).
                 ["name"] = _serverNameCache,
                 ["players"] = _playerManager.PlayerCount,
-                ["engine"] = buildInfo.EngineVersion // MV Launcher can filter based on engine
```

Comment:
I suggest not removing the comment so in six months I can still remember why we did it this way ;)